### PR TITLE
Update block number interval

### PR DESCRIPTION
**Pull request by WinWang21**
Changed block number in mock api back to original and removed unnecessary condition

### CODE REVIEW

File: primerj/src/main/java/org/primer/primerj/utils/BlockUtil.java [hunk 1]
```diff
@@ -169,21 +169,27 @@ public synchronized static Block dowloadSpvBlock() throws Exception {
...
                 log.info("Block: " + block + " ");
-                log.info("Block: interval is " + String.valueOf(PrimerjSettings.INTERVAL));
             }
         } catch (Exception e) {
             e.printStackTrace();
             AbstractApp.notificationService.sendBroadcastGetSpvBlockComplete(false);
             throw e;
         }
+
+        /*
```
Comment (sunnyking) [inline primerj/src/main/java/org/primer/primerj/utils/BlockUtil.java]: Generally should always delete code instead of keeping them in comments.

Comment (WinWang21) [reply primerj/src/main/java/org/primer/primerj/utils/BlockUtil.java]: As suggested, the old code has been deleted.

**Review by sunnyking — APPROVED** [review]

**Review by sunnyking — CHANGES_REQUESTED** [review]
On second thought, the purpose of this code might be to not save every block headers.

**Comment by sunnyking** [comment]
If that's the case consider using 480 as our interval (8 hours for primecoin) and make sure the api always returns the one with height divisible by the interval

**Comment by WinWang21** [comment]
The interval has been changed to 480. Still waiting for changes to be made to the mock api

**Review by sunnyking — APPROVED** [review]

**Pull request merged**